graphql-server/cmd/graphql-server/main.go

```diff
@@ -18,6 +22,22 @@ func main() {
 
 	router := gin.Default()
 
+	if config.Log.Sentry != nil {
+		sentryClientOption := sentry.ClientOptions{
+			Dsn:         config.Log.Sentry.DSN,
+			Environment: config.Log.Sentry.Environment,
+		}
+
+		if err := sentry.Init(sentryClientOption); err != nil {
+			panic(fmt.Sprintf("Failed to initialize sentry: %v\n", err))
+		}
+		router.Use(sentrygin.New(sentrygin.Options{Repanic: true}))
```

Comment:
if you are talking about the `Repanic` flag then it's because sentry would absorb the panic and gin would return 200 if no special handling is done, turning on `Repanic` would allow sentry to repanic the error and gin would then return 500

and yes it logs the request body if it is used in the handler